src/PoolManager.sol

```diff
@@ -80,7 +80,6 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
     using SafeCast for *;
     using Pool for *;
     using Hooks for IHooks;
-    using Position for mapping(bytes32 => Position.State);
```

Comment:
i think youre right. Can you also remove it in the ProxyPoolManager.sol

Reply:
done ✅

Reply:
done ✅

Reply:
@hensha256